requests_ntlm/requests_ntlm.py

```diff
@@ -31,9 +34,11 @@ def __init__(self, username, password, session=None):
                 self.domain = '.'
 
         self.domain = self.domain.upper()
-
         self.password = password
 
+        # Used by other python libraries to sign/seal messages with NTLM methods
+        self.session_security = None
```

Comment:
Aha, cool. =) That's what I didn't understand.

In that case I'd like a nice explanatory comment above that says that this is for use by third-party libraries that would like to take advantage of the negotiated NTLM session.